app/models/item.rb

```diff
@@ -0,0 +1,8 @@
+class Item < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :category
+  belongs_to :price_range
+
+  validates :name, uniqueness: true, presence: true
+  validates :image, :category_id, :description, :price_range_id, presence: true
```

Comment:
Validations for different columns should be on different lines, each with presence true.